components/storage/blockdevices/COMPONENT_FLASHIAP/README.md

```diff
@@ -0,0 +1,18 @@
+# Block Device driver build on top of FlashIAP API
+
+## Warning 
+This driver is **EXPERIMENTAL** and improper usage could kill your board's flash.
```

Comment:
Hmmm, after this move are these still experimental?

Reply:
I guess it is no longer experimental but the improper usage of it may still kill your flash.

Reply:
Oh fair enough. It's unrelated to your PR so we can look at it again after it's in.